test/fixtures/content/page-metadata-jsonld-error.md

```diff
@@ -0,0 +1,11 @@
+# JSON LD Test
+
+This is great.  
+
++----------------------------------------------------+
+| Metadata                                           |
++================+===================================+
+| title          | Home \| Helix Project Boilerplate |
++----------------+-----------------------------------|
+| Json-Ld        | "@context":"http://schema.org"    |
```

Comment:
i would expect to see the payload... eg.

```
    {
      "@context": "https://schema.org/",
      "@type": "Recipe",
      "name": "Party Coffee Cake",
      "author": {
        "@type": "Person",
        "name": "Mary Stone"
      },
      "datePublished": "2018-03-10",
      "description": "This coffee cake is awesome and perfect for parties.",
      "prepTime": "PT20M"
    }
```

Reply:
??

this is the input md for a case, where the JSON is not properly formatted.